src/components/Settings/MobileTab.react.js

```diff
@@ -34,6 +34,13 @@ const CountryCode = styled.div`
   align-items: center;
 `;
 
+const InputText = styled.div`
+  margin-right: 2rem;
+  @media (max-width: 410px) {
+    margin-right: 2px;
```

Comment:
Isn't it redundant?

Reply:
@akshatnitd Without that it will look like this:
![Capture](https://user-images.githubusercontent.com/32234926/62605664-36210c80-b918-11e9-813b-003c74cb2ab1.JPG)